test/deepFreeze/deepFreeze.test.js

```diff
@@ -0,0 +1,6 @@
+const expect = require('expect');
+const deepFreeze = require('./deepFreeze.js');
+
+test('deepFreeze is a Function', () => {
+  expect(deepFreeze).toBeInstanceOf(Function);
+});
```

Comment:
We definitely need at least a few tests, but I will pitch in on these if you have trouble adding them due to time limitations or anything else.

Reply:
Sorry, I had very little time + I had no clue how to test it. 😢

Reply:
I'll add some tests.